test/utils/ViewTest.scala

```diff
@@ -24,13 +24,13 @@ import play.api.i18n.{Lang, Messages, MessagesApi}
 import play.api.mvc.MessagesControllerComponents
 import play.api.test.Helpers
 
-trait ViewTest extends UnitTest with GuiceOneAppPerSuite {
+trait ViewTest extends UnitTest {
 
   implicit lazy val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
   implicit lazy val messages: Messages = messagesApi.preferred(fakeRequest)
   lazy val welshMessages: Messages = messagesApi.preferred(Seq(Lang("cy")))
   implicit lazy val mockMessagesControllerComponents: MessagesControllerComponents = Helpers.stubMessagesControllerComponents()
-  implicit lazy val mockConfig: AppConfig = app.injector.instanceOf[AppConfig]
+  implicit lazy val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
```

Comment:
im not 100% sure why this needs to be made, i think its called mockConfig so we know that were not pulling the real appConfig in our tests. The same as where mockMessagesControllerComponents is of type MessagesControllerComponents. I could be wrong tho 🤷‍♂️

Reply:
Changed it because it isn't actually a mock, it is the AppConfig pulled from the running application in the tests.